packages/tests/functional/basic.ts

```diff
@@ -1,7 +1,8 @@
 describe('Bigcommerce page', () => {
     it('should have the right title', async () => {
+        console.log('Navigating to bigcommerce.com')
         await browser.url('https://www.bigcommerce.com/')
         const title = await browser.getTitle()
         expect(title).toEqual('The Future of Commerce Is Yours | BigCommerce')
     })
-})
\ No newline at end of file
+})
```

Comment:
Sorry I'm a little late to the party, but let's consider linting these files with the same configs to keep the code consistent with the rest of the codebase

Reply:
:+1: